WordPress/src/main/java/org/wordpress/android/viewmodel/posts/PostListItemUiStateHelper.kt

```diff
@@ -153,19 +158,35 @@ class PostListItemUiStateHelper @Inject constructor(
         )
     }
 
-    private fun getDateAndAuthorLabel(
+    private fun getPostInfoLabels(
+        postStatus: PostStatus,
         formattedDate: String,
         displayName: String?,
-        authorFilterSelection: AuthorFilterSelection
-    ): UiString {
-        return when (authorFilterSelection) {
-            EVERYONE -> {
-                val joinedStrings = listOf(formattedDate, displayName).filterNot { TextUtils.isEmpty(it) }
-                        .joinToString(separator = "  ·  ")
-                UiStringText(joinedStrings)
+        authorFilterSelection: AuthorFilterSelection,
+        isSearch: Boolean
+    ): List<UiString> {
+        val uiStrings: MutableList<UiString> = mutableListOf()
+
+        if (!formattedDate.isBlank()) {
+            uiStrings.add(UiStringText(formattedDate))
+        }
+        if (authorFilterSelection == EVERYONE && !displayName.isNullOrBlank()) {
+            uiStrings.add(UiStringText(displayName))
+        }
+
+        if (isSearch) {
+            val postStatusText = when (postStatus) {
+                UNKNOWN -> R.string.unknown
+                PUBLISHED -> R.string.post_status_post_published
+                DRAFT -> R.string.post_status_draft
+                PRIVATE -> R.string.post_status_post_private
+                PENDING -> R.string.post_status_pending_review
+                TRASHED -> R.string.post_status_post_trashed
+                SCHEDULED -> R.string.post_status_post_scheduled
             }
```

Comment:
This doesn't necessarily need to happen in this PR, but it might be useful to create a utility function for this to make it reusable. I guess for now, this is just something to keep in mind.

Reply:
I agree with both statements;). I'll leave it for now and we can move it when we find the need to re-use it.